README.md

```diff
@@ -5,15 +5,16 @@ Library to add ImageView (ImageBadgeView) with a badge like notification count.
 ## Download
 Add to gradle root:
 ```
-buildscript {
-    repositories {
-        jcenter()
-    }
+allprojects {
+	repositories {
+		...
+		maven { url 'https://jitpack.io' }
+	}
 }
 ```
 Download via Gradle:
 ```
-implementation 'com.github.nikartm:image-support:1.0.5'
+implementation 'com.github.sampingantech:Image-Support:1.1.0'
```

Comment:
is it possible to make it lower-case like the origin? @anangkur

Reply:
actually I just copy and paste from jitpack. maybe the explanation from mas @mochadwi can explain this mas @adesamp 

> https://github.com/sampingantech/Image-Support/pull/2#issuecomment-757657498

Reply:
action items? https://github.com/sampingantech/Image-Support/pull/2#issuecomment-757674931